pkg/repository/postgres/dbsqlc/workflows.sql

```diff
@@ -86,7 +86,7 @@ WHERE
     workflows."deletedAt" IS NULL AND
     (
         sqlc.narg('search')::text IS NULL OR
-        workflows.name like concat('%', sqlc.narg('search')::text, '%')
+        workflows.name iLIKE concat('%', sqlc.narg('search')::text, '%')
```

Comment:
nit:

```suggestion
        workflows.name ILIKE CONCAT('%', sqlc.narg('search')::TEXT, '%')
```

we really need to do a better job of consistent casing - lots of queries mix and match 😅